packages/browser/src/node/commands/fill.ts

```diff
@@ -19,6 +19,6 @@ export const fill: UserEventCommand<UserEvent['fill']> = async (
     await browser.$(selector).setValue(text)
   }
   else {
-    throw new TypeError(`Provider "${context.provider.name}" does not support clearing elements`)
```

Comment:
I've noticed that this fallback error was using what seems like a copied text from `clear.ts`. Let's correct the injustice here.